.clang-tidy

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
please remove this file from the commit as it contains user specific configuration

Reply:
Die you see the bugprone-use-after-move comments?

I hoped you use it to fix this errors.

Reply:
I confess I haven't run clang tidy in a long time on the code. But thanks for the hint, will have a look

Reply:
With clang-17 the are much more Warnungen.